ffd_info_exchange/uscis/templates/confirmation-name-change.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <h1 class="headline-text">Thank you for applying for a name change!</h1>
+    <p>Thank you for submitting your name change application &mdash; your confirmation number is <strong>X7Ty51</strong>.</p>
+    <p>For status updates on your application, please call the USCIS customer service center at 1-800-666-5713.</p>
+
+    <div class="button_wrapper">
+        <!-- @todo: Get USWDS 'usa-button-small' class to apply. -->
```

Comment:
@donjo Do you have time to solve this bit?

Context: I changed the markup from a `<button>` to a `<a>` because this page isn't actually a form, but as a result, the `<button class="usa-button-small">` was lost. Applying that class to the `<a>` element doesn't do the trick. I recommend checking the relevant USWDS documentation.